src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
+  }
+
+  return invertedRobot;
```

Comment:
The function correctly returns the 'invertedRobot' object if no duplicate values are found in the input 'robot' object.